### PR TITLE
New image picker plugin

### PR DESCRIPTION
**Pull request by hbaptiste**
New image picker plugin
imagepicker:
       accept: ['*']
       config: []

### CODE REVIEW

File: src/tb/apps/content/components/ContentManager.js [hunk 1]
```diff
@@ -188,6 +193,35 @@ define(
...
+                this.maskMng.mask(oldContentHtml);
+                newContent.getHtml(renderMode).done(function (html) {
+                    jQuery(oldContentHtml).replaceWith(html);
+                    self.computeImages();
+                    oldContentParent.getData("elements").done(function (elements) {
+                        jQuery.each(elements, function (key, data) {
+                            if (data.uid === oldContent.uid) {
+                                elementInfos[key] = { uid: newContent.uid, type: newContent.type };
+                            }
+                            return true;
```
Comment (fkroockmann) [inline src/tb/apps/content/components/ContentManager.js]: It should not be in the condition ?

File: src/tb/component/revisionselector/managers/Save.js [hunk 2]
```diff
@@ -75,6 +75,7 @@ define(
 
                     if (parent !== undefined) {
                         if (element.data('scalar') === true) {
+                            console.log(element);
```
Comment (fkroockmann) [inline src/tb/component/revisionselector/managers/Save.js]: Remove this file from commit plz

File: src/tb/component/medialibrary/templates/layout.tpl [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <div id="library-pane-wrapper" class="bb5-windowpane-wrapper windowpane-treeview">
     <div class="bb5-windowpane-tree ui-layout-pane ui-layout-west">
         <div class="ui-layout-north">
-            <p><strong data-i18n="toolbar.selector.select_folder">Select a topic</strong></p>
+            <p><strong data-i18n="toolbar.selector.select_folder">Select a folder</strong></p>
```
Comment (fkroockmann) [inline src/tb/component/medialibrary/templates/layout.tpl]: Use translate plz

File: src/tb/component/medialibrary/main.js [hunk 4]
```diff
@@ -20,6 +20,7 @@ define(
         'component!rangeselector',
         'jquery',
         'text!tb.component/medialibrary/templates/layout.tpl',
+        'nunjucks',
```
Comment (ndufreche) [inline src/tb/component/medialibrary/main.js]: Use Core\Renderer

**Comment by ndufreche** [comment]
thanks :+1:

**Pull request merged**